packages/cli/src/databases/entities/FeatureConfig.ts

```diff
@@ -0,0 +1,12 @@
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+import type { LdapConfig } from '@/Ldap/types';
+import { jsonColumnType } from './AbstractEntity';
+
+@Entity()
+export class FeatureConfig {
+	@PrimaryColumn()
+	name: string;
+
+	@Column(jsonColumnType)
+	data: LdapConfig;
```

Comment:
I'm assuming that this is a generic class/table to hold any feature related data. so, `data` shouldn't be hard-coded to be `LdapConfig` here.